product_docs/docs/pgd/5/rel_notes/pgd_5.3.0_rel_notes.mdx

```diff
@@ -22,56 +22,56 @@ This version requires the recently released Postgres versions 14.10, 15.4,
 or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
 Server. No such restrictions exist for PostgreSQL Server.
 
-Package managers on Debian, RHEL, and SLES pull in the required EPAS or PGE
-upgrades with an upgrade of Postgres Distributed.
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres Advanced Server or EDB Postgres Extended 
+upgrades with an upgrade of EDB Postgres Distributed.
 !!!
 
 ## Features
 
 | Component | Version | Description  | Addresses |
 |-----------|---------|-------------|-----------|
-| PGD | 5.3.0 | Add support for PostgreSQL 16 server, EDB Postgres Extended Server 16 and EDB Postgres Advanced Server 16. | |
+| PGD | 5.3.0 | Added support for PostgreSQL 16 server, EDB Postgres Extended Server 16, and EDB Postgres Advanced Server 16. | |
 
 ## Enhancements
 
 | Component | Version  |Description | Addresses |
 |-----------|---------|-------------|-----------|
 | Proxy     | 5.3.0   | Added the default service unit file to package | |
-| BDR       | 5.3.0   | Dependencies on EPAS and PGE are now reflected in packages. | |
+| BDR       | 5.3.0   | Dependencies on EDB Postgres Advanced Server or EDB Postgres Extended are now reflected in packages. | |
 
-## Bug Fixes
+## Bug fixes
 
```

Comment:
@djw-m Most of these are not written to style guide and tenses are mixed but I wasn't sure how to fix them. Let's discuss.